packages/components/src/Tooltip/utils/isSemanticElement.spec.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react"
+import { ArrowRightIcon, Button, IconButton } from "../../index"
```

Comment:
Haha I don't think we've ever done this, but I suppose it's okay? Alt:
```suggestion
import { Button, IconButton } from "~components/Button"
import { ArrowRightIcon } from "~components/Icon"
```

Reply:
I like it this way but also don't feel strongly. @gyfchong any opinions?

Reply:
Hmm I think stick with the aliases, incase we move the files around, I don't want to have to find and update individual paths